camviewer_ui_impl.py

```diff
@@ -2587,11 +2596,13 @@ def getConfig(self):
         self.cfg.add("projection", "0")
         self.cfg.add("markers", "0")
         self.cfg.add("dispspec", "0")
+        # self.cfg.add("displayFormat", "%12.8g")
```

Comment:
Remove this.  displayFormat isn't a global option, so you shouldn't put it here.

Reply:
removed